agave_app/AppearanceSettingsWidget.cpp

```diff
@@ -1078,7 +1071,12 @@ QAppearanceSettingsWidget::onNewImage(Scene* scene)
     // init
     this->OnOpacityChanged(i, scene->m_material.m_opacity[i]);
 
+    // get color ramp from scene
+    const ColorRamp& cr = scene->m_material.m_colormap[i];
     QComboBox* gradients = makeGradientCombo();
+    int idx = gradients->findData(QVariant(cr.m_name.c_str()), Qt::UserRole);
+    LOG_DEBUG << "Found gradient " << idx << " (" << cr.m_name << ") for channel " << i;
+    gradients->setCurrentIndex(idx);

```

Comment:
the above code makes sure that on loading from a saved json session, that we initialize the incoming colormaps in the dropdown for each channel

Reply:
remove debug log